packages/1ui/src/components/Sheet/Sheet.tsx

```diff
@@ -0,0 +1,140 @@
+'use client'
```

Comment:
Same -- we can omit these Shad items that are for Next.js specifically. Although, that's an interesting question for us overall. I wonder if we should actually start including/leaving these for builders who use 1ui in Nextjs?

@0xjojikun this is a good thing for us to consider moving forward. We may need to do a pass to re-add these to certain components for use in Next.